apis/nucleus/src/components/listbox/interactions/keyboard-navigation/keyboard-nav-rows.js

```diff
@@ -75,11 +75,10 @@ export default function getRowsKeyboardNavigation({
         }
 
         // Without shift key
-        if (inSelection) {
-          keyboard.focusSelection();
-        } else {
+        if (!keyboard.focusSelection()) {
           currentTarget.blur();
-          keyboard.blur();
+          keyboard.blur(true);
```

Comment:
I am not sure I think going back to focus the container is the right change here.
compared to keep the focuse inside the listbox. (going to cyclic button or search if exist)
but I will accept it as OK.